build

```diff
@@ -195,7 +196,7 @@ $ZIG build \
   --search-prefix "$ROOTDIR/out/$TARGET-$MCPU" \
   -Dflat \
   -Dstatic-llvm \
-  -Doptimize=ReleaseFast \
+  -Doptimize="$ZIG_OPTIMIZE_MODE" \
   -Dstrip \
```

Comment:
One potential issue I see with this PR is this. I would assume that the motivation for distributing e.g. a `ReleaseSafe` build of the compiler is that we can get useful stack traces when something goes wrong on the user's machine. It seems like `-Dstrip` undermines that.

Reply:
good catch

Reply:
but andrew has expressed not wanting to add branching to this script, so that leaves the open question of if the `-Dstrip` should be removed unconditionally and `-Dopotimize` changed to ReleaseSafe always and the option removed in the wake of the acceptance of 15194